Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ExternalDB/ByExternalTable.tsx

```diff
@@ -1,7 +1,7 @@
 //******************************************************************************************************
 //  ByExternalTable.tsx - Gbtc
 //
-//  Copyright � 2019, Grid Protection Alliance.  All Rights Reserved.
+//  Copyright � 2019, Grid Protection Alliance.  All Rights Reserved.
```

Comment:
I think this has something to do with your VS code settings. See if you can fix that to use the correct character set